README.md

```diff
@@ -1,13 +1,34 @@
 # Mongo API Project
 
-Replace this readme with your own information about your project. 
+This project aimed at setting up and storing data in mongoDB, and query that data from an API that I build containing a dataset from spotify with the top songs on the billboard from 2010 - 2019.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+## The features
 
-## The problem
+Endpoints:
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+GET /endpoints - to get
+
+GET /tracks - to get the whole data with all songs
+
+GET /tracks/index - to find a single item with the given index.
+
+GET /tracks/titles/:title - to get a single item with the given title
+
+GET /tracks/artists/artist/:artist - to get a single item with the given artist
```

Comment:
I would just name it tracks/artist/:artist to make it more restful. Since you dont have a route called "artists" where you show all artists i feels unnecessary to have that extra path.